test/test_helper.rb

```diff
@@ -173,7 +173,7 @@ def assert_record(expected, actual, type_name: nil)
   end
 
   def assert_collection(expected, actual, type_name: nil)
-    assert expected.any?
+    raise 'The expected records array can not be empty to assert_collection' unless expected.any?
```

Comment:
```ruby
assert expected.any?, 'The expected records array can not be empty to assert_collection'
```

What do you think?

Reply:
could be, but I don't like it because it is testing the expected collection. It seems weird to have test failures on expected collection to me :thinking:

Reply:
But ok :)